custom_components/healthchecksio/coordinator.py

```
@@ -0,0 +1,74 @@
+        check_url = (
+            f"https://hc-ping.com/{self._check_id}"
+            if not self._self_hosted
+            else f"{self._site_root}/{self._ping_endpoint}/{self._check_id}"
+        )
+
+        try:
+            await self._session.get(
+                check_url,
+                timeout=ClientTimeout(total=10),
+            )
+        except Exception:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)